src/elements/play-pen-header.ts

```diff
@@ -118,9 +117,6 @@ export class PlayPenHeader extends LitElement {
   @query('play-assets-dialog')
   private _assets!: PlayAssetsDialog
 
-  @query('play-export-dialog')
-  private _export!: PlayExportDialog
```

Comment:
I'm confused --- where did this functionality go? this should still be what "Export" does :)

Reply:
Oops I thought you wanted to replace it! I'll add it back in as an option in the dropdown menu with the dialog!